### PR TITLE
Release/20

### PR DESCRIPTION
**Pull request by oguzhanunlu**
This PR

- removes all projects except base-alpine, base-debian, k8s-dataflow
- removes 1 directory per image version approach, to make Dockerfiles source of truth -- git tags will point to specific image versions
- updates CI/CD to introduce stricter checks and linting to comply with best practices
- migrate from Travis to GH Actions
- migrate from Bintray to Docker Hub

### CODE REVIEW

**Comment by oguzhanunlu** [comment]
~~[CI failure](https://travis-ci.org/github/snowplow/snowplow-docker/builds/731582392#L194-L207) is due to current Dockerfiles having issues per hadolint~~

**Review by chuwy — CHANGES_REQUESTED** [review]
`README` is still outdated.

@benjben you might also want to have a look.

**Comment by oguzhanunlu** [comment]
oh, I forgot about README, thanks @chuwy , updating now

**Review by benjben — COMMENTED** [review]
Do we want to keep this repo now that all the components push their own Docker image with CI/CD ?

**Comment by oguzhanunlu** [comment]
We maintain base images' Dockerfiles in this repository, @benjben

**Comment by oguzhanunlu** [comment]
@chuwy @benjben I just wrapped my development on this PR. Could you have another look?

File: README.md [hunk 1]
```diff
@@ -1,187 +1,52 @@
...
-- [S3 Loader][s3]
-- [Iglu Server][iglu-server]
-- [Igluctl][igluctl]
-- [Piinguin Server][piinguin-server]
-- [EmrEtlRunner][emr-etl-runner]
+- base-alpine
+- base-debian
+- k8s-dataflow
 
 They are published in the [`snowplow-docker-registry.bintray.io`][registry] docker registry.
```
Comment (chuwy) [inline README.md]: Wrong.

**Review by chuwy — APPROVED** [review]
Looks great. Awesome idea with Hadolint.

Two things should be changed though:

1. Still a lot of mentions of Bintray in README
2. I couldn't spot the difference between images, e.g. when should I use `base-debian` and when `base-alpine`

Otherwise - 👍

**Review by benjben — APPROVED** [review]
Awesome work @oguzhanunlu !!

Agreed with Anton that it would be good to know the difference between the 2 base images.

I would not put the versions in `docker pull` in the README, that's easily forgotten after an update.

**Review by chuwy — APPROVED** [review]

**Pull request merged**